src/source/parserutil.d

```diff
@@ -8,26 +8,32 @@ auto match(Lexer, TokenType)(ref Lexer lexer, TokenType type) {
 		return token;
 	}
 
-	import std.conv, std.string;
-	auto error = token.type == TokenType.Invalid
-		? token.error.toString(lexer.context)
-		: format!"expected '%s', got '%s'."(to!string(type),
-		                                    to!string(token.type));
-
-	import source.exception;
-	throw new CompileException(token.location, error);
+	import std.conv;
+	throw unexpectedTokenError(lexer, to!string(type));
 }
 
 auto unexpectedTokenError(Lexer)(ref Lexer lexer, string expected) {
 	auto token = lexer.front;
 
 	import std.conv, std.string;
```

Comment:
Why is this imported instead of std.format?

Reply:
https://dlang.org/phobos/std_conv.html#to